index.css

```diff
@@ -119,53 +129,57 @@ main {
 }
 
 /* project section  */
+
 .project__section {
-  width: 372;
-  display: flex;
+  display: grid;
+  gap: 30px;
   background: #fff;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  border: 0 2px 0 2px solid #ff6b00;
-  padding-bottom: 100px;
+  padding: 0 0 80px 0;
+}
+
+.recent__works {
+  display: flex;
+  flex-direction: column;
+  padding: 0 24px 0 24px;
 }
 
 .project__title {
   display: flex;
+  align-items: center;
   flex-direction: column;
-  margin-top: 80px;
-  width: 327px;
-  height: 80px;
   font-family: 'Crete Round', sans-serif;
   font-weight: 200;
-  font-size: 40px;
-  margin-left: 30px;
+  font-size: 32px;
   color: #1c1a19;
+  padding: 50px 24px 50px 24px;
 }
 
-.project__title-bar {
-  margin-top: 20px;
+.project__title img {
+  padding: 10%;
+  width: 100%;
 }
 
 .project__image {
-  display: flex;
-  margin-top: 50px;
-  width: 327px;
-  height: 237px;
-  left: 24px;
-  background-image: url('./images/placeholder.png');
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  background-image: url(../icons_copy/ImgPlaceholder.svg);
```

Comment:
- [ ] Kindly fix the path on your background image. It should have only one dot. Like ./ at the beginning, in this way it will be displayed on the browser